lib/test_prof.rb

```diff
@@ -91,10 +92,15 @@ class Configuration
     def initialize
       @output = $stdout
       @color = true
-      @output_dir = "tmp"
+      @output_dir = "tmp/test_prof"
```

Comment:
Shouldn't we ensure that that dir exists?

I think, we can move `mkdir_p` to `TestProf.artefact_path` method to create directory only when necessary.